redkyn/canvas/exceptions.py

```diff
@@ -0,0 +1,38 @@
+from redkyn.exceptions import RedkynError
+from requests.exceptions import HTTPError
+
+def raiseAuthenticationFailed(err: HTTPError):
+    if err.response.status_code != 401:
+        return
+
+    raise AuthenticationFailed(err)
+
+class AuthenticationFailed(RedkynError):
+    pass
+
+
+def raiseCourseNotFound(err: HTTPError):
+    if err.response.status_code != 404:
+        return
+
+    raise CourseNotFound(err)
+
+class CourseNotFound(RedkynError):
+    pass
+
+
+def raiseNameResolutionFailed(err: HTTPError):
+    if err.response is not None:
+        return
+
+    if err.request is not None:
+        return
+
+    print(str(err))
```

Comment:
Do you want this to print if it says "Name or service not known?"

Reply:
Whoops, I think this is actually old debug code :upside_down_face: